### PR TITLE
- hermitpack implemented

### PR DESCRIPTION
**Pull request by AlexMdle**

### CODE REVIEW

File: src/main/java/thePackmaster/cards/hermitpack/DeadOrAlive.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        if (this.energyOnUse < EnergyPanel.totalCount) {
+            this.energyOnUse = EnergyPanel.totalCount;
+        }
+
+        int num = energyOnUse;
+
+        if (p.hasRelic("Chemical X")) {
+            num += 2;
+            p.getRelic("Chemical X").flash();
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/hermitpack/DeadOrAlive.java:34]: XCost logic in card's use is a nono

File: src/main/java/thePackmaster/cards/hermitpack/Headshot.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    private static final int UPGRADE_PLUS_DMG = 2;
+
+    public Headshot() {
+        super(ID, 1, CardType.ATTACK, CardRarity.COMMON, CardTarget.ENEMY);
+        baseDamage = DAMAGE;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        int dam = this.damage;
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/hermitpack/Headshot.java:27]: What's happening here?

File: src/main/java/thePackmaster/cards/hermitpack/Headshot.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    @Override
+    public void calculateCardDamage(AbstractMonster mo)
+    {
+        super.calculateCardDamage(mo);
+
+        if (isDeadOnPos()) {
+            int base_dam = this.damage;
+            this.damage += base_dam;
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/hermitpack/Headshot.java:47]: Why not just multiply damage by 2 `damage *= 2;`

File: src/main/java/thePackmaster/cards/hermitpack/Quickdraw.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    public void use(AbstractPlayer p, AbstractMonster m) {
+        AbstractDungeon.actionManager.addToBottom(
+                new DamageAction(m, new DamageInfo(p, damage, damageTypeForTurn), AbstractGameAction.AttackEffect.BLUNT_LIGHT)
+        );
+        this.addToBot(new DrawCardAction(magicNumber));
+
+        DrawReductionPower pow = new DrawReductionPower(p,DRAWLESS);
+        ReflectionHacks.setPrivate(pow,DrawReductionPower.class,"justApplied",false);
+
+        this.addToBot(new ApplyPowerAction(p, p, pow, DRAWLESS));
```
Comment (erasels) [inline src/main/java/thePackmaster/cards/hermitpack/Quickdraw.java:40]: When justApplied is false, this gets reduced atEndOfRound, DRAWLESS is 1, so this gets removed at the end of the round before you even draw your cards for the next round and does nothing?

**Review by erasels — CHANGES_REQUESTED** [review]

File: src/main/java/thePackmaster/cards/hermitpack/Headshot.java (hunk 3 above)
Comment (AlexMdle) [reply src/main/java/thePackmaster/cards/hermitpack/Headshot.java:47]: Remnant from when Snipe was a factor, shouldn't matter either way now, but will shorten.

File: src/main/java/thePackmaster/cards/hermitpack/Quickdraw.java (hunk 4 above)
Comment (AlexMdle) [reply src/main/java/thePackmaster/cards/hermitpack/Quickdraw.java:40]: I don't remember the particulars, but it has to do with the strange way draw reduction debuff works. Without this, it does not get reduced at the start of your turn until a turn later.

File: src/main/java/thePackmaster/cards/hermitpack/Headshot.java (hunk 2 above)
Comment (AlexMdle) [reply src/main/java/thePackmaster/cards/hermitpack/Headshot.java:27]: No idea.

**Review by erasels — APPROVED** [review]

**Pull request merged**